### ISSUE TITLE
Add warning to footer of outgoing messages letting public bodies know their replies may get redacted

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/whatdotheyknow-theme#867]
Public bodies are sometimes confused when email addresses they include in responses get redacted prior to the response being made available to the requester or published. 

This causes a problem:

* When an email address is requested
* When a public body seeks to refer a case to another public body
* When a public body asks a requester to contact them directly
* When a public body advises sending internal review requests to a particular email address

There have on occasion been exchanges between confused FOI officers and users who cannot understand why the email address sent, cannot be seen. Given advice on what is happening is one click away, and linked from the text which replaces the removed email address, this usually impacts those struggling to use the service. 

The outgoing message footer currently states

>
>-------------------------------------------------------------------
>
>Please use this email address for all replies to this request:
>request-[xxxxx-xxxxxx]@whatdotheyknow.com
>
>Is [xxxxxxx] the wrong address for Freedom of Information requests to [Public Body Name]? If so, please contact us using this form:
>https://www.whatdotheyknow.com/change_request/new?body=[public_body_name]
>
>Disclaimer: This message and any reply that you make will be published on the internet. Our privacy and copyright policies:
>https://www.whatdotheyknow.com/help/officers
>
>For more detailed guidance on safely disclosing information, read the latest advice from the ICO:
>https://www.whatdotheyknow.com/help/ico-guidance-for-authorities
>
>Please note that in some cases publication of requests and responses will be delayed.
>
>If you find this service useful as an FOI officer, please ask your web manager to link to us from your organisation's FOI page.
>
>
>-------------------------------------------------------------------

That's already long and unwieldy, but we could add a line noting something like:

>The WhatDoTheyKnow.com system will seek to redact email addresses and mobile phone numbers from the plain text of correspondence prior to publication or making the response available to the requester. 

I think this is too wordy.

However this issue does cause correspondence with the team and does confuse users and public bodies so I am ticketing it. 

[Edit: Typo fixed on -> one]

**Comment by garethrees** [comment]
-1 to a footer this long.

I think we should reduce it to something more like:

> This message and any reply that you make will be published on the internet.
> Please use request-[xxxxx-xxxxxx]@whatdotheyknow.com for all correspondence to this request.
> Read more about replying to FOIs made through WDTK: https://www.whatdotheyknow.com/help/officers

**Comment by RichardTaylor** [comment]
The redaction of emails is already mentioned at 

https://www.whatdotheyknow.com/help/officers#mobiles

so our advice is available if the link - which is shown both to the user on the request thread, and to the officer via the footer,  is followed. 

I agree the footer is already too long, and I would support shortening it. 

This ticket was raised though because public body officers, and users, weren't following the links. 

Another route to addressing the underlying issue is:

* https://github.com/mysociety/alaveteli/issues/10

**Comment by garethrees** [comment]
closing in favour of https://github.com/mysociety/whatdotheyknow-theme/issues/1115
